### PR TITLE
Handle module countries

### PR DESCRIPTION
**Pull request by KangOl**

### CODE REVIEW

**Comment by KangOl** [comment]
upgradeci retry with always only hr_payroll in master

**Comment by KangOl** [comment]
add temporary tests

Allow verfication (on upgradeCI) that the previous commits works as expected.

 - test that new auto install modules without countries are still
       installed
 - test that new auto install modules with countries are correctly
       filtered

upgradeci retry with always only hr_payroll hr_holiday+hr_contract in master

**Comment by KangOl** [comment]
upgradeci retry with hr_homeworking+calendar

**Comment by KangOl** [comment]
upgradeci retry with hr_homeworking+calendar

File: hr_contract/tests/test_123.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+@change_version("saas~17.4")
+class TestPR123(UpgradeCase):
+    def prepare(self):
+        return True
+
+    def check(self, data):
+        get = self.env["ir.module.module"]._get
+
+        self.assertEqual(get("hr_holiday_contract").state, "installed")
```
Comment (aj-fuentes) [inline hr_contract/tests/test_123.py:12]: The test is not running? (missing in `__init__`)
```suggestion
        self.assertEqual(get("hr_holidays_contract").state, "installed")
```

Comment (KangOl) [reply hr_contract/tests/test_123.py]: No, the error is more stupid than that 🤦

**Comment by KangOl** [comment]
upgradeci retry with always only hr_payroll hr_holidays+hr_contract hr_homeworking+calendar in master

**Review by aj-fuentes — APPROVED** [review]
LGTM

**Review by xavieralt — APPROVED** [review]

**Comment by KangOl** [comment]
@robodoo rebase-ff r+

**Pull request closed without merging**